01_standalone_examples/databricks-ci-cd/README.md

```diff
@@ -0,0 +1,222 @@
+# lakeFS-samples-ci-cd
+
+Start by ⭐️ starring [lakeFS open source](https://go.lakefs.io/oreilly-course) project.
+
+**This sample captures a collection of Databricks notebooks and GitHub Action code that demonstrate how to run Databricks ETL jobs in an isolated environment by using lakeFS.**
+
+## Prerequisites
+* lakeFS installed and running on a server or in the cloud. If you don't have lakeFS already running then either use [lakeFS Cloud](https://demo.lakefs.io/) which provides free lakeFS server on-demand with a single click or refer to [lakeFS Quickstart](https://docs.lakefs.io/quickstart/) doc.
+* Databricks server with the ability to run compute clusters on top of it. 
+* Configure your Databricks cluster to use lakeFS Hadoop file system. Read this blog [Databricks and lakeFS Integration: Step-by-Step Configuration Tutorial](https://lakefs.io/blog/databricks-lakefs-integration-tutorial/) or [lakeFS documentation](https://docs.lakefs.io/integrations/spark.html#lakefs-hadoop-filesystem) for the configuration.
+* Permissions to manage the cluster configuration, including adding libraries. 
+* GitHub account. 
+
+## Setup
+
+1. Create [Databricks personal access token](https://docs.databricks.com/en/dev-tools/auth/pat.html).
+
+
+1. Create Databricks secret scope e.g. **demos** or use an existing secret scope. Add following secrets in that secret scope by following [Secret management docs](https://docs.databricks.com/en/security/secrets/index.html): 
```

Comment:
Maybe we should provide here the example of how to do that?

i.e. 
```
databricks secrets put-secret --json '{
  "scope": "demos",
  "key": "lakefs_access_key_id",
  "string_value": "AKIAIOSFOLKFSSAMPLES"
}'

databricks secrets put-secret --json '{
  "scope": "demos",
  "key": "lakefs_secret_access_key",
  "string_value": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
}'
```
WDYT?

Reply:
Thanks for the example. Added it.